src/object/cli_obj.c

```
@@ -5816,7 +5817,7 @@ dc_obj_update(tse_task_t *task, struct dtx_epoch *epoch, uint32_t map_ver,
 
 	rc = obj_update_shards_get(obj, args, map_ver, obj_auxi, &shard, &shard_cnt);
 	if (rc != 0) {
-		D_ERROR(DF_OID" get update shards failure %d\n", DP_OID(obj->cob_md.omd_id), rc);
+		D_INFO(DF_OID " get update shards failure %d\n", DP_OID(obj->cob_md.omd_id), rc);
```

I think WARN might be more appropriate but we are also potentially ignoring errors here.   We typically use D_CDEBUG for such cases to make sure the error is retryable or such before we lower its log level

---

done